source/languages/en/riak/dev/taste-of-riak/golang.md

```diff
@@ -0,0 +1,429 @@
+---
+title: "Taste of Riak: Go"
+project: riak
+version: 1.4.0+
```

Comment:
Yep! Thank you for asking, because I haven't been paying attention to that part of the front matter.